frontend/js/pages/ListSettledBattles.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import React, { Component } from 'react';
+
+import TableHeader from '../components/TableHeader';
+import TableSettledRow from '../components/TableSettledRow';
+
+class ListSettledBattles extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tableHeader: ['', 'Battle Number', 'Created', 'Trainers', 'You Won'],
+      battles: [],
+    };
+  }
+
+  async componentDidMount() {
+    axios.get(`/api/battles/settled`).then((response) => {
```

Comment:
It's a good practice to have a different folder to store API calls instead of making them directly in the component. How you organize it is up to you, but this [blogpost](https://blog.danawoodman.com/articles/organizing-react-projects#organizing-non-react-code) is a great example!